Modules/Package.swift

```diff
@@ -283,13 +294,12 @@ enum XcodeSupport {
                     .product(name: "KeychainAccess", package: "KeychainAccess"),
                 ]
             ),
-            .xcodeTarget(
-                XcodeTargetNames.storage,
-                dependencies: ["Codegen", "WooFoundation"]
-            ),
             .xcodeTarget(
                 XcodeTargetNames.storageTests,
-                dependencies: ["TestKit", XcodeTargetNames.storage.asDependency]
+                dependencies: [
+                    "Storage",
+                    "TestKit"
```

Comment:
Why the move to plain strings for these, out of interest?

Reply:
No particular reason. I used the DRY names for the Xcode support targets, but didn't put anything in place to do the same once the frameworks became packages.

I'd like to do so at some point, but maybe after everything has finished moving.